molecule/default/python.yml

```
@@ -4,6 +4,7 @@
   become: yes
   become_method: sudo
   roles:
-    - pip
-    - python
-    - remove_python2
+    - install_pip2: true
+      role: pip
+    - install_python2: true
+      role: python
```

Aren't role variables [normally handled like this](https://docs.ansible.com/ansible/latest/user_guide/playbooks_reuse_roles.html#roles-keyword):
```suggestion
    - role: pip
      vars:
        install_pip2: true
    - role: python
      vars:
        install_python2: true
```

---

Ah, sorry, I was just going with how it's done  in https://github.com/cisagov/cyhy_amis/blob/d368af9ab4c7e7e5481c325c04c431b2a50e2bbb/packer/ansible/playbook.yml#L42-L51 since it was fresh on my mind. I can change it if we'd prefer this way (and changed what I linked accordingly).

---

I prefer the way I suggested; even better might be to use `ansible.builtin.import_role` as I think that is the preferred way to do it now.

---

If we go the `ansible.builtin.import_role` route I think that change should be made in [cisagov/skeleton-ansible-role](https://github.com/cisagov/skeleton-ansible-role) and trickle down. I'll update to use your suggested syntax instead for this PR.